server/incident/incident.go

```diff
@@ -102,7 +102,8 @@ type StatusPost struct {
 }
 
 type UpdateOptions struct {
-	ActiveStage *int `json:"active_stage"`
+	CommanderUserID *string `json:"commander_user_id"`
+	ActiveStage     *int    `json:"active_stage"`
```

Comment:
We can revert this change, as [the current implementation](https://github.com/mattermost/mattermost-plugin-incident-management/blob/master/server/api/incidents.go#L140-L168) only changes the active stage.
```suggestion
	ActiveStage *int `json:"active_stage"`
```